test/lib/mayaUsd/render/mayaToHydra/cpp/testIsolateSelect.py

```diff
@@ -58,9 +57,9 @@ class TestIsolateSelect(mtohUtils.MayaHydraBaseTestCase):
 
     @classmethod
     def setUpClass(cls):
-        if Usd.GetVersion() >= (0, 24, 11):
-            cls.imageVersion = 'usd_2411+'
         super(TestIsolateSelect, cls).setUpClass()
+        if cls._usdVersion >= (0, 24, 11):
+            cls.imageVersion = 'usd_2411+'
```

Comment:
What I had in mind was actually to move the imageVersion attribute itself to `MayaHydraBaseTestCase`, but I can see how not all tests might want to use that as their image version(s)